src/test/kotlin/no/nav/syfo/oppfolgingstilfelle/bit/OppfolgingstilfelleBitSpek.kt

```diff
@@ -1,24 +1,17 @@
-package no.nav.syfo.oppfolgingstilfelle
+package no.nav.syfo.oppfolgingstilfelle.bit
 
-import io.mockk.mockk
-import no.nav.syfo.oppfolgingstilfelle.bit.OppfolgingstilfelleBit
-import no.nav.syfo.oppfolgingstilfelle.bit.OppfolgingstilfelleBitService
 import no.nav.syfo.oppfolgingstilfelle.bit.Tag.*
 import no.nav.syfo.util.defaultZoneOffset
 import org.amshove.kluent.shouldBeEqualTo
 import org.spekframework.spek2.Spek
 import org.spekframework.spek2.style.specification.describe
 import testhelper.generator.generateOppfolgingstilfelleBit
-import java.time.*
-
-class OppfolgingstilfelleServiceSpek : Spek({
-    describe(OppfolgingstilfelleServiceSpek::class.java.simpleName) {
-
-        val oppfolgingstilfelleBitService: OppfolgingstilfelleBitService = mockk()
-        val oppfolgingstilfelleService = OppfolgingstilfelleService(
-            oppfolgingstilfelleBitService = oppfolgingstilfelleBitService,
-        )
+import java.time.LocalDate
+import java.time.OffsetDateTime
+import java.time.Period
 
+class OppfolgingstilfelleBitSpek : Spek({
+    describe(OppfolgingstilfelleBitSpek::class.java.simpleName) {
```

Comment:
Er det en idé å si noe om hva man tester her i stedet for å gjenta klassenavnet? "Unit tests for creating oppfølgingstilfelle from biter", elns?